examples/simple-username-database/README.md

```diff
@@ -0,0 +1,71 @@
+
+
+# Supabase Project Documentation
+
+This documentation provides an overview of a JavaScript project that utilizes Supabase for the database. The project allows users to enter their username.It is an open-source project created by [preetsuthar17](https://github.com/preetsuthar17) and can be found on [GitHub](https://github.com/preetsuthar17).
+
+## Prerequisites
+> Setup your Supabase SQL before continuing this work.
+>For more information about using Supabase, refer to the official Supabase documentation: [Supabase Documentation](https://supabase.io/docs).
+
+
+Before running the project, please make sure you have Node.js installed on your machine. If you don't have Node.js, you can download and install it from the official website: [Node.js Website](https://nodejs.org).
+
+## Installation
+
+To get started with the project, follow these steps:
+
+1. Clone the repository from GitHub:
+   ```
+   git clone https://github.com/preetsuthar17/supabase-gemo.git
```

Comment:
Here as well 👆

Reply:
After merging pull request we can change that link to this repository.